docs/plugin-development-quickstart.md

```diff
@@ -0,0 +1,31 @@
+---
+author: Randolph
+---
+
+# Quickstart for Plugin Development
+
+## 1. Start a plugin file
+
+input following code to start a plugin file
+
+> npx gulp new-pkg
+
+...(a brief introduction about the files generated)
+
+## 2. How to inject ui
+
+1. change definitions in SNSAdaptor to add entry
+2. what is metadata and how to decode and encode metadata  
```

Comment:
```suggestion
2. what is metadata and how to decode and encode metadata
```

Reply:
Should be trimmed by editor config plugin. @Randolph314 Recommend you to have a EditorConfig plugin install in you IDE/Editor.

https://github.com/DimensionDev/Maskbook/blob/2ab3ce2e76414eb6f95ac8cbf7202f6244b947fd/.editorconfig#L7

Reply:
BTW, there are some typos.